src/core/include/iDynTree/Core/EigenHelpers.h

```diff
@@ -73,6 +73,43 @@ inline Eigen::Map<Eigen::VectorXd> toEigen(iDynTree::Span<double> vec)
 {
     return Eigen::Map<Eigen::VectorXd>(vec.data(),vec.size());
 }
+
+inline Eigen::Map<const Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>,
+                  0,
+                  Eigen::Stride<Eigen::Dynamic, Eigen::Dynamic>>
+toEigen(const MatrixView<const double>& mat)
+{
+    using MatrixRowMajor = Eigen::Matrix<double,Eigen::Dynamic,Eigen::Dynamic,Eigen::RowMajor>;
+
+    // This is a trick required to see a ColMajor matrix as a RowMajor matrix.
+    const int innerStride = (mat.storageOrder() == StorageOrder::ColMajor) ? mat.rows() : 1;
+    const int outherStride = (mat.storageOrder() ==StorageOrder::ColMajor) ? 1 : mat.cols();
```

Comment:
``outher`` goes without the h. The same below.

Reply:
Ops

Reply:
Done in c173e11622e51cf232319915ca8b57232480b425